edit-post/api/index.js

```diff
@@ -0,0 +1,4 @@
+export {
+	renderSidebar,
+	getSidebarSettings,
```

Comment:
The exposure of getSidebarSettings was a remnant of some debugging. I'll remove it from the API. I cannot think of a reason to leave it exposed. When someone thinks of a use case it can be exposed again.